static-analysis/teammates-eslint.yml

```diff
@@ -7,9 +7,6 @@ env:
 extends:
     - airbnb-base
 
-parserOptions:
-    ecmaVersion: 2015
-
 rules:
     comma-dangle: ["error", {
```

Comment:
We don't need to override this anymore.

Reply:
I have inserted trailing commas in functions to overcome lint errors after removing this rule but we seem to be using `babel-plugin-syntax-trailing-function-commas` so things should work fine right ?